modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/SelectVocabularies.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayAlert from '@clayui/alert';
+import {ClayDualListBox} from '@clayui/form';
+import React, {useState} from 'react';
+
+export default function ({
+	availableVocabularyNames,
+	currentVocabularyNames,
+	namespace,
+}) {
+	const vocabularies = [currentVocabularyNames, availableVocabularyNames];
+
+	const data = vocabularies.map((inner) =>
```

Comment:
Just suggestion: Since this is only being used here, it may make sense to eliminate the use of allocating a variable for this case.

```js
const data = [currentVocabularyNames, availableVocabularyNames].map((inner) =>
```

Reply:
👍